lib/src/services.dart

```diff
@@ -0,0 +1,102 @@
+import "dart:async";
+import 'package:chopper/chopper.dart';
+
+class OmniServices with ChopperServiceMixin {
+  OmniServices([ChopperClient client]) {
+    this.client = client;
+  }
+
+  final definitionType = OmniServices;
+
+  /// Get all available markets.
+  Future<Response> fetchMarkets() {
+    final url = '/api/v2/markets';
```

Comment:
It would be good to have a private method that removes the need for repeating `/api/v2` everywhere. What do you think about `this.pathFor('markets')`?

Reply:
Also, notice that this is not a URL, it's a path, the URL includes the domain.